source/_integrations/bitvavo.markdown

```diff
@@ -0,0 +1,35 @@
+---
+title: Bitvavo
+description: Keep track of cryptocurrencies.
+ha_category:
+  - Finance
+ha_release: 2021.4
+ha_iot_class: Cloud Polling
+ha_codeowners:
+  - '@cyberjunky'
+ha_domain: bitvavo
+ha_config_flow: true
+---
+
+With the Bitvavo integration, you can keep track of cryptocurrencies from the [Bitvavo](https://bitvavo.com) exchange.
+
+## Requirements
+
+Setup requires an API Key and Secret Key from Bitvavo with at least 'Read' permissions. Browse to [API Settings](https://account.bitvavo.com/user/api) to generate your API Key and Secret Key.
+
+<div class='note warning'>
+
+A word of caution: It is highly recommended to use the IP whitelist, and specify the lowest level of permissions you need. Using an API will bypass 2FA, keep your keys safe!
```

Comment:
> the lowest level of permissions you need

What would that lowest level of permissions be for this integration?

Reply:
The warning is more generic, the integration only needs "Read Only" rights, which is the lowest level for Bitvavo.
Shall I change the text?

Reply:
I think this is better, as "you need" was a bit like: "I don't know what I need?" :)

Good change 👍